src/main/java/jenkins/scm/api/mixin/ChangeRequestSCMHead.java

```diff
@@ -47,6 +47,30 @@
     @NonNull
     String getId();
 
+    /**
+     * Returns the {@link ChangeRequestCheckoutStrategy} of this {@link ChangeRequestSCMHead}.
+     *
+     * @return the {@link ChangeRequestCheckoutStrategy}.
```

Comment:
```suggestion
     * @return the {@link ChangeRequestCheckoutStrategy}.
     * @since TODO
```

Reply:
Actually, I do not very understand the process of the annotation `since`. Who and when will fill the real version for replacing `TODO`. Please give me a hint if you know it. Thanks. But it's good advice.